### ISSUE TITLE
High contrast mode issue and some suggestion.

### ISSUE THREAD

**Issue by askasys-code** [binarynonsense/comic-book-reader#181]
Hello, first of all, I want to thank you for this magnificent project; it is the best reader available on PC. I have tried them all, even the paid ones, and ACBR beats them all in versatility and simplicity.

Now, let me show you a non-problem. Or rather, my case is a truly rare usage scenario.
Basically, I use a high-contrast theme, and I would like this application to start by default with `--disable-features=ForcedColors`, without me having to create a shortcut. The high-contrast theme is not well compatible with ACBR, so it would be better to bypass it.

I also take this opportunity to ask you a few more things:

- Support for conversion and reading of JPEG-XL format.
- Hardware decoding, if possible in the future, or some way to speed up reading, because some comics use very heavy images, and ACBR tends to be slow in reading files. For example, on Android, I use ReadEra, and it is very fast; it loads everything instantly.
- Continuous scroll mode from top to bottom, like ReadEra, as I believe it is the best viewing mode for comics on a tablet. ACBR can be installed on Windows tablet devices, such as Surface or Legion GO and Rog Ally.

**Comment by binarynonsense** [comment]
Hi,

- I have no experience with high contrast themes so I hadn't thought about it being an issue, but from a quick search it seems it should be easy to make the CSS ignore forced colors settings, so I'll try to add that as a default in the next release.
- Supporting JPEG-XL in the reader and converter is not an option right now as neither Chromium/Electron nor Sharp seem to do it. If that changes in the future I'll revisit this.
- I believe GPU acceleration is enabled by default in Chromium/Electron if supported but I have to look into it some more. I'll see if there's a way to check and to enable it or not but, that said, I think ACBR's speed reading pages is mainly limited by the speed of the file decompression (for cbr, cbz and cb7 files) and that's the best I've managed with the libraries I use so far, so I don't think I'll be able to improve much on this area, at least for now (there are some ideas I want to experiment with in the future, but don't know when and if they'll lead to any improvements). 
- Adding continuous mode is discussed in this open issue: https://github.com/binarynonsense/comic-book-reader/issues/86

In summary: looks like the high contrast issue itself is the only thing I think I'll be able to implement/fix for now, sorry the suggestions seem to be most likely "out of scope" at least for the time being, for one reason or another. If that changes at any point I'll let you know.

Thanks for your kind words and feature requests!

**Comment by binarynonsense** [comment]
I just uploaded a new beta, [v3.8.0-beta1](https://github.com/binarynonsense/comic-book-reader/releases/tag/v3.8.0-beta1), that disables the forced colors when using a high contrast theme. Seems to work from a quick test I made but, if you can, let me know if you give it a try to make sure it does.

**Comment by binarynonsense** [comment]
I just uploaded a new stable version, [v3.8.0](https://github.com/binarynonsense/comic-book-reader/releases/tag/v3.8.0), that includes this so I'm going to close the issue, but let me know if you give it a try at some point.
